src/Conn.cc

```diff
@@ -113,8 +114,8 @@ Connection::Connection(NetSessions* s, const ConnIDKey& k, double t, const ConnI
 	++current_connections;
 	++total_connections;
 
-	TimerMgr::Tag* tag = current_iosrc->GetCurrentTag();
-	conn_timer_mgr = tag ? new TimerMgr::Tag(*tag) : 0;
+	std::string* tag = current_iosrc->GetCurrentTag();
```

Comment:
This change is a bit unfortunate as it drops the semantics of what that string is. How about renaming the original `Tag` type into something that doesn't conflict (e.g., `TimerMgr::ID`, `TimerMgr::Label`, etc.)